packages/core/admin/admin/src/content-manager/pages/EditView/components/Header.tsx

```diff
@@ -476,7 +477,14 @@ const DeleteAction: DocumentActionComponent = ({ documentId, model, collectionTy
         </Flex>
       ),
       onConfirm: async () => {
-        setSubmitting(true);
+        /**
+         * If we have a match, we're in the list view
+         * and therefore not in a form and shouldn't be
+         * trying to set the submitting value.
+         */
+        if (!match) {
```

Comment:
i think the code would be clearer if you named the var something like listPathMatch. Because here when guessing what match is for it's 50-50 between form and list paths